src/SFA.DAS.EmployerCommitmentsV2.Web/Validators/WhoWillEnterTheDetailsViewModelValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Validators
+{
+    public class WhoWillEnterTheDetailsViewModelValidator : AbstractValidator<WhoWillEnterTheDetailsViewModel>
+    {
+        public WhoWillEnterTheDetailsViewModelValidator()
+        {
+            RuleFor(r => r.AccountHashedId).NotEmpty();
```

Comment:
What error message is returned to the user for these top 3 rules? Does it make sense to the user as these are not something they have submitted?

Reply:
Removed as discussed